releasenotes/notes/custom-serializers-qpy-0097ab79f239fcfc.yaml

```diff
@@ -0,0 +1,42 @@
+---
+features:
+  - |
+    Added a new kwarg, ``metadata_serializer`` to the
+    :func:`.qpy_serialization.dump` function for specifying a custom
+    serialization function for use when serializing the
+    :attr`.QuantumCircuit.metadata` attribute. By default the
+    :func:`~qiskit.circuit.qpy_serialization.dump` function will attempt to
```

Comment:
for laziness: you can do both `~` and `.`, so `` :func:`~.qpy_serialization.dump` `` does the lookup with the `.` semantics, and displays it using the `~` truncation semantics (so it just shows `dump`).  No change needed here, though.